src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertEnumCertificatesInStore.cs

```diff
@@ -8,12 +8,12 @@ internal static partial class Interop
 {
     internal static partial class Crypt32
     {
-#if DLLIMPORTGENERATOR_ENABLED
+#if UNITTEST
```

Comment:
I thought we were going with explicitly setting `EnableDllImportGenerator` in test projects that used these shared files